src/Razor/src/Microsoft.AspNetCore.Razor.OmniSharpPlugin/DefaultProjectChangePublisher.cs

```diff
@@ -155,41 +155,59 @@ internal void ProjectManager_Changed(object sender, OmniSharpProjectChangeEventA
                     // These changes can come in bursts so we don't want to overload the publishing system. Therefore,
                     // we enqueue publishes and then publish the latest project after a delay.
 
-                    EnqueuePublish(args.Newer);
+                    if (args.Newer.ProjectWorkspaceState != null)
```

Comment:
Just curious why we need this?

Reply:
Because `null` == we haven't analyzed the project yet. It's basically a "semi" initialized project which isn't as interesting to us. It's what we do in VS anyhow. I could see us going back on this decision in the future if we see fit. In the end it effectively results in a few extra seconds of "no IntelliSense" which is a trade for telling the user they temporarily have a bajillion "unknown" component errors. 😄